backend/src/main/kotlin/demo/acme/model/Store.kt

```diff
@@ -0,0 +1,30 @@
+package demo.acme.model
+
+import javax.persistence.*
+
+@Entity
+data class Store(
+    @Id @GeneratedValue(strategy=GenerationType.SEQUENCE) var id:Long,
+    var code:String,
+    @Column(length = 5000) var description:String,
+    var name:String, 
+    var openingDate: String,
+    var storeType: String,
+    @OneToMany(cascade = [CascadeType.ALL]) var seasons:MutableList<Season>,
```

Comment:
How would you efficiently fix these duplications?

![imagem](https://user-images.githubusercontent.com/39516190/127896202-c2cabf6c-187f-47f1-bf04-d69cd0884b47.png)

Reply:
Changed the collection to a Set instead